server/app/database/data/spells.json

```diff
@@ -24509,7 +24509,7 @@
                       "en": "1 cell from center of trap",
                       "fr": "1 case d'éloignement",
                       "de": "1 cell from center of trap",
-                      "es": "1 cell from center of trap",
+                      "es": "a 1 casilla del centro de la trampa",
```

Comment:
This is used here ![Fragmentation Trap](https://i.imgur.com/IJwYZvj.png) - you think the first letter should stay lowercase?